components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2815,12 +2827,15 @@ impl GenSnapTask {
     where
         EK: KvEngine,
     {
+        self.index
+            .store(last_applied_state.applied_index, Ordering::Relaxed);
```

Comment:
Suggest to use `SeqCst` instead, we usually use `Relaxed` only when for metrics or protected by lock.